packages/core/src/utilities/getClosestImageId.ts

```diff
@@ -43,9 +43,6 @@ export default function getClosestImageId(
   // If not, the view is not aligned with the image plane, so return early.
   const dotProduct = vec3.dot(kVector, viewPlaneNormal);
   if (Math.abs(dotProduct) < 1 - EPSILON) {
-    console.debug(
```

Comment:
Can you move to log.info as this is a useful message to have, even if it isn't commonly used?